### DISCUSSION TITLE
How to get the latest version of the tiddlyclip plug in

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#55] [Q&A] [answered]
This is the version of tiddlyclip  installed in my wiki 

<img width="579" alt="image" src="https://user-images.githubusercontent.com/67494083/148672862-c7c8e2b2-d2da-4ffd-8afc-9a64b133b9c4.png">

But when I checked in the the GitHub repo, I could see that the latest version is 01.17.16

<img width="742" alt="image" src="https://user-images.githubusercontent.com/67494083/148672929-39456b2d-3d6a-4e36-80af-b54d61f80090.png">

How to download the latest version? I am using a standalone wiki, not the node.js one

**Comment by buggyj** [comment]
HI arunnbabu81, 
you are the first to ask about getting the updated version, my assumption is that not many are using tiddlyclip! 

I have made many improvements over the years to tiddlyclip - I use it all the time to record and organize information that I find on the web. 

I use the updated version of the tiddlyclip plugin with the latest version of the browser addon. Currently I have am loading the latest browser addon (version 1.6)  from my local disk - in order for others to use it I need to update my level of firefox security so that I can get mozilla to package the addon. I will try and do this in the next couple of days so you can  install it easily. After than I will put the new pluggin version here.


cheers
BJ

**Comment by arunnbabu81** [comment]
Tiddlyclip is very useful when I want to add text and images from webpages to my wiki. I guess many people must be using it. And I think its important to read the documentation to use it properly. 

Eagerly awaiting for the release of the updated versions of the add-on and plug in. It would be nice if you could document  what all changes have been made to the plug in since the version 01.14.2

**Comment by arunnbabu81** [comment]
When to expect the updated plug in

**Comment by arunnbabu81** [comment]
This is the my wiki with tw 5.2.1

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/67494083/149799472-08d5457f-f9c0-46d6-9848-58de7f50859c.png">

and this wiki is based on TW 5.1.23

<img width="989" alt="image" src="https://user-images.githubusercontent.com/67494083/149799628-bcecc5e2-06a7-44fb-b525-163f35cf717e.png">


All the newly created rules doesn't appear in the Firefox context menu for the wiki based on 5.2.1

This issue and the issue regarding [snaps](https://github.com/buggyj/tiddlyclip/issues/69#issue-1103189019) are preventing me from using tiddlyclip fully in my wiki

**Comment by buggyj** [comment]
I am waiting for mozilla to package the new version - should be ready for testing tomorrow

**Comment by buggyj** [comment, marked as answer]
There is now a candidate release - you can try if you like. I am testing it today and will release tomorrow if all is ok

Addon: https://github.com/buggyj/tiddlyclip/releases/download/0.1.8/tiddlyclip-0.1.8-fx.xpi
plugin: https://github.com/buggyj/tiddlyclip-plugin/releases/download/5.1.17/tiddlyclip.tid

**Comment by arunnbabu81** [comment]
I tried both the add on and plug in. 

- I can confirm that the [issue](https://github.com/buggyj/tiddlyclip/issues/70#issue-1106190680) of tids not appearing in context menu is solved now. 
- But the snaps are still not taken in the selected portions as told in this [issue](https://github.com/buggyj/tiddlyclip/issues/69#issue-1103189019). Is it due to something specific to Mac OS or my system (I use an external monitor to mirror my Mac).

I have noticed that there are many new tiddlers in the new plug in as compared to the old 01.14.2 version I was using. Does that mean there are many more things to explore in tiddlyclip!!

Big thanks to you for doing all this updates for us. It has been great to learn so much for you. Thank you.

**Comment by buggyj** [comment]
There are many ways to use tiddlyclip which I have not mentioned. I use tiddlyclip with a number of my other plugins to re-arrange web content, not only text but video and audio content.

**Comment by buggyj** [comment]
One feature I find very useful is to add selections of tags to clips - this is achieved by adding a list of tags to another tiddlyclip config file  called TiddlyClipOpts like so:
![image](https://user-images.githubusercontent.com/2855795/150007604-be46ac98-f0f2-43c7-93fd-631e22a2e597.png)



So the list of tags appear under the tiddlyclip dropdown, checking the boxes means that will be added to the next tiddler created by a clip or snap etc.
